src/components/Event/EventList.tsx

```diff
@@ -0,0 +1,21 @@
+import EventListItem, { iEventItem } from "./EventItem";
+
+interface EventListProps {
+  events: Array<iEventItem>;
+}
+
+export const EventList = (props: EventListProps) => {
+  return (
+    <>
```

Comment:
<></> not required here. <div></div> is the parent element and it's required for layout/style.